src/Resources/Gift/Gift.ts

```diff
@@ -1,13 +1,14 @@
 import AbstractResource from '../AbstractResource';
+import { HttpClientConfig, HttpClientResponse } from '../../Interfaces/HttpClient';
 
 export default class Gift extends AbstractResource {
     initialise(): void {
         this.resourcePath = '/redeem-gift';
         this.defaultAccess = 'member';
     }
 
-    public redeem: Function = this.createMethodFromPartialSpec({
+    public redeem: (...args: HttpClientConfig) => HttpClientResponse = this.createMethodFromPartialSpec({
         method: 'POST',
-        path: '/{code}',
```

Comment:
did you mean to remove this?

Reply:
Nah this was in the master branch. I originally branched off the master branch because I thought someone said that we're switching the default to master. This is a change from master - Johns change I believe.